src/python/pants/backend/experimental/python/lockfile.py

```diff
@@ -80,13 +88,28 @@ def from_tool(cls, subsystem: PythonToolRequirementsBase) -> PythonLockfileReque
             description=f"Generate lockfile for {subsystem.options_scope}",
         )
 
+    @property
+    def hex_digest(self) -> str:
```

Comment:
As mentioned in Slack, it would be excellent to unit test this. Try out a couple input combinations and make sure they equal the hardcoded expected hash.

Reply:
Done